benchmarking/chain-sync/analyse-logs.sh

```diff
@@ -3,17 +3,17 @@
 NETWORK=${1:-mainnet}
 
 # get first block copy time
-LOGFILE=`ls -1 state-node-${NETWORK}/node-0-*.log | head -3`
-FIRSTSLOT=`grep -e '.*:cardano.node.ChainDB:.*TraceCopyToImmDBEvent.CopiedBlockToImmDB.*"tip":"[a-z0-9]*@' $LOGFILE | sed -ne 's/^.* \[\([0-9-]\+\) \([0-9:.]\+\) UTC\] .*"tip":"[a-z0-9]*@\([0-9]\+\)".*/"\1 \2";\3/p; q;'`
+LOGFILE_JSON=`ls -1 state-node-${NETWORK}/node-0-*.json | head -2`
+FIRSTSLOT=`jq --slurp 'map (select (.data.event.kind == "TraceCopyToImmDBEvent.CopiedBlockToImmDB")) | .[0] | "\(.at);\(.data.tip | sub (".*@(?<slot>.*)$"; "\(.slot)"))"' $LOGFILE_JSON | xargs echo | sed 's/\([0-9]\)T\([0-9]\)/\1 \2/; s/\(20[0-9][0-9]\)/"\1/; s/Z/"/'`
```

Comment:
this does not yet work.

Reply:
Hmm, it did work locally, interesting.

Reply:
@CodiePP, where were you seeing it as not working -- it did both locally and on CI: https://buildkite.com/input-output-hk/cardano-node-nightly-sync/builds/26#0cd7b168-25cd-498e-a957-a62dde1222af